Simulator/Vertices/NG911/All911Vertices.h

```diff
@@ -138,14 +138,19 @@ class All911Vertices : public AllVertices {
    /// True if the call was abandoned
    vector<vector<bool>> wasAbandonedHistory_;
```

Comment:
Is this a variable that will be needed on the GPU? If so, make it `vector<vector<unsigned char>>` and treat its elements as having `bool` values (i.e., using `true` and `false`).